.github/workflows/ci.yaml

```diff
@@ -20,6 +20,9 @@ jobs:
     - name: Install golint
       run: go install golang.org/x/lint/golint@latest
       if: ${{ always() }}
+    - uses: dominikh/staticcheck-action@v1.3.0
+      with:
+        version: "2022.1.3"
```

Comment:
Should we just exclude this and use latest?

Reply:
that works, updated